bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerChecker.java

```diff
@@ -267,6 +306,12 @@ public void operationComplete(int rc, LedgerFragment result) {
      */
     public void checkLedger(final LedgerHandle lh,
                             final GenericCallback<Set<LedgerFragment>> cb) {
+        checkLedger(lh, cb, 100L);
```

Comment:
interesting, this setting will enforce checking all the entries, no?

@govind-menon can you confirm what is the purpose of this?

Reply:
Yes, 100 will checking all entries. 
In the first commit, it was "0", and in third commit it was turned into 100.